packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -111,7 +111,7 @@ exports.buildForeignReferenceMap = ({
                 }
                 foreignReferenceMap[v.sys.id].push({
                   name: `${contentTypeItemId}___NODE`,
-                  id: entryItem.sys.id,
+                  id: entryItem.sys.id
```

Comment:
Please run "npm run format" from the root of the repo

Reply:
ooh so that's the trick! ok thanks